examples/CMakeLists.txt

```diff
@@ -1,17 +0,0 @@
-add_executable(ccl_sample_baryons ccl_sample_baryons.c)
```

Comment:
I wonder if the problem has to do with having deleted this. Maybe the root CMakeLists.txt should be changed too.

Reply:
I remove the ref to it from the main one. Should work now I think.